Watchman.DomainModel/Warns/Queries/GetWarnEventsQuery.cs

```diff
@@ -8,12 +8,14 @@ namespace Watchman.DomainModel.Warns.Queries
     public class GetWarnEventsQuery : IQuery<GetWarnEventsQueryResults>
     {
         public ulong ServerId { get; }
-        public ulong UserId { get; }
+        public ulong ReceiverId { get; }
+        public DateTime From { get; set; }
```

Comment:
lepiej od razu From i To

+mamy coś takiego jak PaginationQuery -> które od razu ma takie rzeczy jak From, To, Limit itd

Reply:
zrobiłem From i To bo PaginationQuery ma trochę inne pola